uint16 vector_size
task_w_ports[] missions
